sample-app/src/App.tsx

```diff
@@ -1,29 +1,42 @@
-import VerticalSearchPage from './pages/VerticalSearchPage';
 import UniversalSearchPage from './pages/UniversalSearchPage';
+import FAQsPage from './pages/FAQsPage';
+import EventsPage from './pages/EventsPage';
 import PageRouter from './PageRouter';
 import StandardLayout from './pages/StandardLayout';
 import { AnswersHeadlessProvider } from '@yext/answers-headless-react';
 import { universalResultsConfig } from './universalResultsConfig';
+import JobsPage from './pages/JobsPage';
+import LocationsPage from './pages/LocationsPage';
 
 const routes = [
   {
     path: '/',
     exact: true,
     page: <UniversalSearchPage universalResultsConfig={universalResultsConfig} />
   },
-  ...Object.keys(universalResultsConfig).map(key => {
-    return {
-      path: `/${key}`,
-      page: <VerticalSearchPage verticalKey={key} />
-    }
-  })
+  {
+    path: '/faqs',
+    page: <FAQsPage verticalKey='faqs'/>
+  },
+  {
+    path: '/events',
+    page: <EventsPage verticalKey='events'/>
+  },
+  {
+    path: '/locations',
+    page: <LocationsPage verticalKey='locations' />
+  },
+  {
+    path: '/jobs',
+    page: <JobsPage verticalKey='jobs' />
+  }
 ];
 
 export default function App() {
   return (
     <AnswersHeadlessProvider
-      apiKey='2d8c550071a64ea23e263118a2b0680b'
-      experienceKey='slanswers'
+      apiKey='3517add824e992916861b76e456724d9'
+      experienceKey='answers-js-docs'
       locale='en'
       verticalKey='people'
```

Comment:
Do we still want a verticalKey of `people` here?

Reply:
I don't see a reason to keep it, so I'll remove it